src/style.css

```diff
@@ -0,0 +1,81 @@
+* {
+  box-sizing: border-box;
+}
+
+.input {
+  font-family: inherit;
+  width: 100%;
+  font-size: 15px;
+  height: 70px;
+  display: block;
+  padding-left: 65px;
+  margin-bottom: 20px;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.input-small {
+  font-family: inherit;
+  width: 100%;
+  font-size: 13px;
+  height: 40px;
+  display: block;
+  padding-left: 40px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
```

Comment:
You duplicate a lot of styles for both inputs. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2-nd point